### ISSUE TITLE
Questions about this paper

### ISSUE THREAD

**Issue by yanglanq** [KennethLangedal/GNN-MWVC#1]
I have a question about the GNN training stage.

For the MWVC problem, the number of vertices not in the solution is seriously more than the number of vertices in the solution which causes an imbalance in model training. How can I solve this problem? 

Looking forward to your reply.

Best wishes.

**Comment by KennethLangedal** [comment]
That's not necessarily the case, it can be either extreme. For instance, a clique requires |V|-1 vertices in the solution, while a star only requires one. So one option is to select graphs where the classes are more equally balanced.
Another option is to weigh the classes differently in your loss function. I believe cross-entropy loss is typically implemented with this feature.

I hope that helps

**Comment by yanglanq** [comment]
Thanks for your reply, I'll try it.
